test/proxy_funds.js

```diff
@@ -0,0 +1,165 @@
+const { assertRevert } = require('./helpers/assertThrow')
```

Comment:
Can we rename the test file to something a bit more descriptive like `proxy_recover_funds`?